cli/server.go

```diff
@@ -26,11 +31,25 @@ var ServerCmd = cli.Command{
 }
 
 // startEmmyServer configures and starts the gRPC server at the desired port
-func startEmmyServer(port int, certPath, keyPath string) error {
-	srv, err := server.NewProtocolServer(certPath, keyPath)
+func startEmmyServer(port int, certPath, keyPath, logfilePath, loglevel string) error {
```

Comment:
`logfilePath, loglevel` -> `logFilePath, logLevel` (camelCase variable names)

Reply:
Oops. Fixed.

Reply:
Is it? 😄